internal/platform/gin/auditlog/middleware_test.go

```diff
@@ -77,7 +77,9 @@ func TestMiddleware(t *testing.T) {
 
 		middleware := Middleware(driver, WithClock(clock), WithUserIDExtractor(userIDExtractor))
 
+		gin.SetMode(gin.ReleaseMode)
```

Comment:
**Question**: what is the relevance of this change?
(Checked the issue and the patch release, but I might have overlooked this, because I haven't noticed it.)

Reply:
It is only for disabling a warning flood when there are failing tests as per [this issue](https://github.com/gin-gonic/gin/issues/1346) (this line is also present in other tests in the codebase).

Reply:
That's ok.